docs/real-world-examples/ccfraud.md

```diff
@@ -1,49 +1,87 @@
 
-:warning: Proceed to the next step only once the pipeline finishes.
+## Add your kaggle credentials to the workspace keyvault
+
+We propose to run a job in the AzureML workspace that will unpack this demo dataset into each of your silos.
+
+Since the dataset is from Kaggle, we will first store your username/password in the workspace keyvault. This will allow the job to download the dataset from Kaggle directly.
+
+1. In your workspace resource group (provisioned in the previous step), open "Access Policies" tab in the newly keyvault.
+
+2. Select "Select all" right under "Secret Management Operations" and press "Next".
+
+3. Click "Create" button in the top. Lookup currently logged in user (using user id or an email), select it and press "Next". 
+
+4. Press "Next" and "Create" in the next screens.
+
+    We are now able to create a secret in the keyvault.
+
+5. Open the "Secrets" tab. Create two plain text secrets:
+    
+    - **kaggleusername** - specifies your kaggle user name
+    - **kagglekey** - this is API key that can be obtained from your profile on the kaggle
+
+## Run a job to download and store the dataset in each silo
+
+This can all be performed with ease using a data provisioning pipeline. To run it follow these steps:
+
+1. In this repository, navigate in the folder `examples/pipelines/utils/upload_data/`
+
+2. If you are not using the quickstart setup, adjust the config file  `config.yaml` in `examples/pipelines/utils/upload_data/` to match your setup.
+
+3. Submit the experiment by running:
+
+   ```bash
+   python ./examples/pipelines/utils/upload_data/submit.py --submit --example CCFRAUD --workspace_name "<workspace-name>" --resource_group "<resource-group-name>" --subscription_id "<subscription-id>"
+   ```
+
+    :star: you can simplify this command by entering your workspace details in the file `config.yaml` in this same directory.
+
+:warning: Proceed to the next step only once the pipeline completes. This pipeline will create data in 3 distinct locations.
+
+## Run the demo experiment
+
+1. If you are not using the quickstart setup, adjust the config file  `config.yaml` in `examples/pipelines/ccfraud/` to match your setup.
+
+2. Submit the FL experiment by running:
+
+   ```bash
+   python ./examples/pipelines/ccfraud/submit.py --submit --workspace_name "<workspace-name>" --resource_group "<resource-group-name>" --subscription_id "<subscription-id>"
+   ```
+
+    :star: you can simplify this command by entering your workspace details in the file `config.yaml` in this same directory.
+
+## Beyond the experiment
+
+This sample experiment provides multiple models you can try.
 
-### Model choice
 Please update [**config**](../../examples/pipelines/ccfraud/config.yaml), field `model_name` in the `training_parameters` section, to reflect desired model to be trained, options include: SimpleLinear, SimpleLSTM, SimpleVAE
```

Comment:
This sentence is pretty hard to parse. It could use a couple articles, being split after "trained", and a dot at the end.

Reply:
Sure, made another proposal.